src/docs/reference/modules/Media/README.md

```diff
@@ -475,8 +475,35 @@ When the query string is signed with a token any width, height value may be used
     Tokens are only available from the [Preview Feed](../../../getting-started/preview-package-source)
     Prior to this the width or height values are limited to `16`, `32`, `50`, `100`, `160`, `240`, `480`, `600`, `1024`, `2048`.
 
+## Media Indexing
+
+Media can be optionally indexed for search as well if files are referenced via Media Fields. The following data can be indexed for each file referenced from a Media Field:
```

Comment:
Can we add apostrophe in `Media Field`?

Reply:
What do you mean? I don't see a need for an apostrophe here ("files are referenced via Media Fields" is in plural and shouldn't be "files are referenced via Media Field's").

Reply:
It's not the matter of plural, I just want markdown highlight the text ;)

Reply:
It's not the matter of plural, I just want markdown highlight the text ;)

Reply:
I really don't understand what you mean. This?

```suggestion
Media can be optionally indexed for search as well if files are referenced via `Media Field`s. The following data can be indexed for each file referenced from a `Media Field`:
```

However, these are not technical identifiers but English nouns. What I'd denote with backticks would be "MediaField", as in the name of the class.